source/solvers/mf_navier_stokes_operators.cc

```diff
@@ -718,7 +744,7 @@ NavierStokesSUPGPSPGOperator<dim, number>::local_evaluate_residual(
       integrator.evaluate(EvaluationFlags::values | EvaluationFlags::gradients |
                           EvaluationFlags::hessians);
 
-      auto h = integrator.read_cell_data(this->get_element_size());
+      // auto h = integrator.read_cell_data(this->get_element_size());
```

Comment:
What is the issue? What about creating an issue and collecting there the TODOs?

Reply:
We have a TODO list in the MF Lethe Project in the projects section of github. I will give you access to it.